ops/solana/solana.go

```diff
@@ -295,16 +317,20 @@ func (d Deployer) InitOCR(keys []opsChainlink.NodeKeys) error {
 
 	fmt.Println("Setting up OCR Feed:")
 
-	fmt.Println("Begin set offchain config...")
-	err := d.gauntlet.ExecCommand(
-		"ocr2:begin_offchain_config",
+	fmt.Println("Begin offchain proposal...")
+	if err := d.gauntlet.ExecCommand(
+		"ocr2:create_proposal",
 		d.gauntlet.Flag("network", d.network),
 		d.gauntlet.Flag("version", "2"),
-		d.gauntlet.Flag("state", d.Account[OCRFeed]),
-	)
+	); err != nil {
+		return errors.Wrap(err, "create proposal failed")
```

Comment:
```suggestion
		return errors.Wrap(err, "'ocr2:create_proposal' call failed")
```

If this is surfaced through an original error, maybe even is unnecessary to wrap.